connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorUtils.java

```diff
@@ -98,19 +112,51 @@ static Pattern compilePatternList(String fields) {
         return compilePatternList(Arrays.asList(fields.split("\\W*,\\W*")));
     }
 
-    static void createCompactedTopic(String topicName, short partitions, short replicationFactor, Map<String, Object> adminProps) {
+    static void createCompactedTopic(String topicName, short partitions, short replicationFactor, ForwardingAdmin forwardingAdmin) {
         NewTopic topicDescription = TopicAdmin.defineTopic(topicName).
                 compacted().
                 partitions(partitions).
                 replicationFactor(replicationFactor).
                 build();
 
-        try (TopicAdmin admin = new TopicAdmin(adminProps)) {
-            admin.createTopics(topicDescription);
+        CreateTopicsOptions args = new CreateTopicsOptions().validateOnly(false);
```

Comment:
`validateOnly` default to `false` so maybe we can remove this and call the `createTopics()` method with a single argument. WDYT?

Reply:
Digging a bit more I see you reused the same logic we already have `TopicAdmin.createOrFindTopics()` so maybe we can keep it.

Reply:
You're right I kept it as `TopicAdmin.createOrFindTopics()` is using `validateOnly` and am trying to keep the same logic while am switching the code to use ForwardingAdmin instead.